guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>If your data is stored on a cloud store, you can register it as a DataStore using the <code>add_cloudstore</code> function. This function can register cloud store as a DataStore for Azure Data Lake Storage, Amazon, Alibaba, or Azure. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>